ee/vellum_ee/workflows/display/nodes/vellum/subworkflow_deployment_node.py

```diff
@@ -29,7 +29,8 @@ def serialize(
                 input_name=variable_name,
                 value=variable_value,
                 display_context=display_context,
-                input_id=self.node_input_ids_by_name.get(variable_name),
+                input_id=self.node_input_ids_by_name.get(f"subworkflow_inputs.{variable_name}")
```

Comment:
[nit] 
```suggestion
                input_id=self.node_input_ids_by_name.get(f"{SubworkflowDeploymentNode.subworkflow_inputs.name}.{variable_name}")
```

This makes the dependencies on this attribute discrete instead of hardcoded

Reply:
will update!